<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player</class>
 <widget class="QWidget" name="player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#player {
    background-color: #2e2e2e;
    color: #ffffff;
    font-family: &quot;Segoe UI&quot;, sans-serif;
}</string>
  </property>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: #ffffff;
}

QPushButton:hover {
    background-color: #5a5a5a;
    border-radius: 5px;
}

QPushButton:pressed {
    background-color: #0078d7;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoNext"/>
   </property>
  </widget>
  <widget class="QPushButton" name="lastButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: #ffffff;
}

QPushButton:hover {
    background-color: #5a5a5a;
    border-radius: 5px;
}

QPushButton:pressed {
    background-color: #0078d7;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
   </property>
  </widget>
  <widget class="QPushButton" name="stopPauseButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: #ffffff;
}

QPushButton:hover {
    background-color: #5a5a5a;
    border-radius: 5px;
}

QPushButton:pressed {
    background-color: #0078d7;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
   </property>
  </widget>
  <widget class="QPushButton" name="muteButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: #ffffff;
}

QPushButton:hover {
    background-color: #5a5a5a;
    border-radius: 5px;
}

QPushButton:pressed {
    background-color: #0078d7;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::AudioVolumeHigh"/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="coverLayout">
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="coverLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>311</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="dataLayout">
    <item>
     <widget class="QPlainTextEdit" name="dataPlainText">
      <property name="plainText">
       <string>Artist:
Album:
Song:
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="soundSlider">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
    background: #404040;
    height: 6px;
    border-radius: 3px;
}

QSlider::handle:horizontal {
    background: #ffffff;
    border: 1px solid #777777;
    width: 14px;
    height: 14px;
    margin: -4px 0;
    border-radius: 7px;
}

QSlider::sub-page:horizontal {
    background: #0078d7;
    border-radius: 3px;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar#progressBar {
    border: 2px solid #5a5a5a;
    border-radius: 5px;
    background-color: #404040;
    text-align: center;
}

QProgressBar#progressBar::chunk {
    background-color: #0078d7; /* Azul */
    border-radius: 3px;
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    color: #ffffff;
}

QPushButton:hover {
    background-color: #5a5a5a;
    border-radius: 5px;
}

QPushButton:pressed {
    background-color: #0078d7;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
